lib/gollum-lib/filter/bibtex.rb

```diff
@@ -0,0 +1,56 @@
+begin
+  require 'bibtex'
+  require 'citeproc'
+  require 'csl'
+  require 'csl/styles'
+rescue LoadError => error
+  puts "Error trying to require an optional gem for BibTeX parsing: #{error.to_s}"
```

Comment:
Yeah, I also wasn't sure. I always find it a bit strange to rescue an exception and just ignore it, but in this case I think we should maybe remove it: I keep seeing the error when running the tests. :)